frontend/awx/views/schedules/wizard/ScheduleAddWizard.tsx

```diff
@@ -0,0 +1,85 @@
+import { DateTime } from 'luxon';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+import {
+  PageHeader,
+  PageLayout,
+  PageNotImplemented,
+  PageWizard,
+  PageWizardStep,
+} from '../../../../../framework';
+import { useGetPageUrl } from '../../../../../framework/PageNavigation/useGetPageUrl';
+import { dateToInputDateTime } from '../../../../../framework/utils/dateTimeHelpers';
+import { AwxRoute } from '../../../main/AwxRoutes';
+import { ScheduleInputs } from '../components/ScheduleInputs';
+import { ScheduleFormWizard } from '../types';
+import { awxErrorAdapter } from '../../../common/adapters/awxErrorAdapter';
+import { useGetTimezones } from '../hooks/useGetTimezones';
+
+export function CreateSchedule() {
+  const { t } = useTranslation();
+  const getPageUrl = useGetPageUrl();
+  const now = DateTime.now();
+
+  const closestQuarterHour: DateTime = DateTime.fromMillis(
+    Math.ceil(now.toMillis() / 900000) * 900000
+  );
+  const navigate = useNavigate();
+
+  const [currentDate, time]: string[] = dateToInputDateTime(closestQuarterHour.toISO() as string);
+  const handleSubmit = async (formValues: ScheduleFormWizard) => {
+    const { unified_job_template_object, launch_config, prompt } = formValues;
+    const promptValues = prompt;
+
+    if (promptValues) {
+      if (unified_job_template_object && 'organization' in unified_job_template_object) {
+        promptValues.organization = unified_job_template_object.organization ?? null;
+      }
+      if (launch_config) {
+        promptValues.original = {
+          launch_config,
+        };
+      }
+    }
+
+    await Promise.resolve();
+  };
+
+  const onCancel = () => navigate(-1);
+
+  const { timeZones, links } = useGetTimezones();
+
+  const steps: PageWizardStep[] = [
+    {
+      id: 'typeStep',
+      label: t('Node details'),
```

Comment:
According to the mockups this should be `Details`

Reply:
Good catch